core/templates/domain/question/QuestionObjectFactorySpec.ts

```diff
@@ -46,9 +46,11 @@ import { UpgradedServices } from 'services/UpgradedServices';
 // ^^^ This block is to be removed.
 
 require('domain/question/QuestionObjectFactory.ts');
+require('domain/state/StateObjectFactory.ts');
 
 describe('Question object factory', function() {
   var QuestionObjectFactory = null;
+  var stateObjectFactory = null;
   var _sampleQuestion = null;
   var _sampleQuestionBackendDict = null;
```

Comment:
Remove the underscore.

Reply:
@marianazangrossi Hey I didn't actually write those variables. In the future should I change other people's variables, or work with them?

Reply:
Yes! It's ok to change code that you didn't write. Every time you see a code that is not following a pattern or not seems to be right, please fix it.

Reply:
Got it changing it

Reply:
Done